Assignment_1/q9.py

```diff
@@ -0,0 +1,14 @@
+married = bool(input("is the driver married "))
```

Comment:
Three inputs should be required here: `Marital Status: [married, unmarried]`, `Gender: [male, female]` and `Age`.